components/Footer/index.js

```diff
@@ -0,0 +1,357 @@
+    marginBottom: 36
+  }),
+  topRow: css({
+    display: 'flex',
+    justifyContent: 'space-between',
+    marginBottom: 36
+  }),
+  middleRow: css({
+    display: 'flex',
+    justifyContent: 'space-between',
+    flexDirection: 'column',
+    marginBottom: 36,
+    [mediaQueries.mUp]: {
+      flexDirection: 'row-reverse'
+    }
+  }),
+  nav: css({
+    display: 'flex',
+    justifyContent: 'space-between',
+    flexDirection: 'column',
+    maxWidth: 650,
+    flex: 1,
+    [mediaQueries.mUp]: {
+      flexDirection: 'row'
+    }
+  }),
+  addressColumn: css({
+    marginRight: 100
+  }),
+  since: css({
+    ...fontStyles.sansSerifRegular12,
+    display: 'none',
+    marginLeft: 8,
+    [mediaQueries.mUp]: {
+      display: 'inline-block'
+    }
+  }),
+  navList: css({
+    display: 'flex',
+    flexDirection: 'row',
+    listStyleType: 'none',
+    flexWrap: 'wrap',
+    padding: 0,
+    margin: 0,
+    marginBottom: 32,
+    [mediaQueries.mUp]: {
+      flexDirection: 'column',
+      marginBottom: 0
+    },
+    '& li': {
+      ...fontStyles.sansSerifRegular16,
+      marginTop: 8,
+      marginRight: 22,
+      [mediaQueries.mUp]: {
+        marginRight: 0
+      }
+    },
+    '& li:first-child': {
+      ...fontStyles.sansSerifRegular12,
+      marginTop: 0,
+      width: '100%',
+      [mediaQueries.mUp]: {
+        width: 'inherit'
+      }
+    }
+  })
+}
+
+const Footer = ({ t, me, signOut, inNativeIOSApp, isOnMarketingPage }) => {
+  const [colorScheme] = useColorContext()
+  const navLinkStyle = useMemo(
+    () =>
+      css({
+        ...fontStyles.sansSerifRegular16,
+        color: colorScheme.getCSSColor('text'),
+        textDecoration: 'none',
+        '@media (hover)': {
+          ':hover': {
+            textDecoration: 'underline'
+          }
+        }
+      }),
+    [colorScheme]
+  )
+
+  const FooterNavLink = props => <Link prefetch={false} {...props} />
+
+  const router = useRouter()
+
+  return (
+    <div {...styles.bg}>
+      <div {...styles.content}>
+        <div {...styles.topRow}>
+          <FooterNavLink href='/'>
+            <a>
+              {isOnMarketingPage ? (
+                <div style={{ width: 23, display: 'inline-block' }}>
+                  <BrandMark {...colorScheme.set('fill', 'text')} />
+                </div>
+              ) : (
+                <Logo {...colorScheme.set('fill', 'text')} height={20} />
+              )}
+              <span {...colorScheme.set('color', 'textSoft')} {...styles.since}>
+                {t('footer/since')}
+              </span>
+            </a>
+          </FooterNavLink>
+
+          <SocialLinks />
+        </div>
+        <hr {...styles.hr} {...colorScheme.set('borderColor', 'divider')} />
+        <div {...styles.middleRow}>
+          <div {...styles.nav}>
+            <ul {...styles.navList}>
+              <li {...colorScheme.set('color', 'disabled')}>
+                {me ? 'Meine Republik' : 'Mitglied werden'}
```

Comment:
use `t` :) And IMO always call it `Meine Republik` because it should also contain a login link and not just joining